plugins/validate.json

```diff
@@ -178,36 +178,36 @@
         "6febce3201c5b050981568312dd8c86f": "ViSimulator.dll"
     },
     "CS-Script (C# intellisense)": {
-        "fef69e2da992b9cc51adf2c67aadee1f": "CSScriptNpp.x64.dll",
-        "a4f736cb6dca06a3571ae9c2d47d0a30": "CSScriptNpp\\CSScriptIntellisense.dll",
+        "99dcb57ca9047be3f89ce2022d61aa4b": "CSScriptNpp.dll",
+        "1c947576119e1fc201d5f77fc049c3e5": "CSScriptNpp\\CSScriptIntellisense.dll",
         "ab0dd2d5c2267a5e3323a3f17f0c9265": "CSScriptNpp\\CSScriptLibrary.dll",
-        "6abede625059a6a1c52175391b46dd5a": "CSScriptNpp\\CSScriptNpp.dll",
+        "8758632c78a23e28e5ff10ceae15ca82": "CSScriptNpp\\CSScriptNpp.dll",
```

Comment:
This one is also found now and probably causing the trouble with 1.7.12.

Reply:
No this one seems correct. I just recalculated MD5, it is the same. If you are wondering about the package having two dlls with the same name then it is what is required. Unfortunately. I had to no choice but to make this duplication because I had to rename the original root `CSScriptNpp.x86.dll`:

``` 
├── plugins
│ ├── CSScriptNpp.x86.dll --> CSScriptNpp.dll
│ ├── CSScriptNpp
│ │ ├── CSScriptNpp.dll
│ │ ├── .....
```
This is exactly what I was complaining about.

And you are absolutely right I accidentally modified `visimulator` plugin. It was almost midnight and I just lost the concentration... Sorry about that. Will fix is shortly.